common/config/env/beta.env

```diff
@@ -0,0 +1 @@
+FLAVOR=beta
```

Comment:
`FLAVOR` is too generic a name for an envvar.

Can we name this `COMMUNICATION_REACT_FLAVOR`?